x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/user_actions_get_users.ts

```diff
@@ -167,204 +169,256 @@ export default ({ getService }: FtrProviderContext): void => {
         );
       });
 
-      it('returns only the creator of the case', async () => {
-        const postedCase = await createCase(
-          supertestWithoutAuth,
-          getPostCaseRequest(),
-          200,
-          null,
-          superUserHeaders
-        );
+      describe('schema', () => {
+        it('ensures that the schema of security plugin is as expected', async () => {
+          const avatar = {
+            initials: 'ES',
+            color: '#6092C0',
+            imageUrl: 'my-image',
+          };
+
+          // update user profile avatar
+          await updateUserProfileAvatar({
+            supertest,
+            req: avatar,
+            headers: superUserHeaders,
+          });
 
-        const { participants, assignees, unassignedUsers, reporter } = await getCaseUsers({
-          caseId: postedCase.id,
-          supertest,
+          const res = await bulkGetUserProfiles({
+            supertest: supertestWithoutAuth,
+            req: {
+              uids: [superUserProfile.uid],
+              dataPath: 'avatar',
+            },
+          });
+
+          const userProfile = res[0];
+
+          try {
+            const userToValidate = {
+              user: {
+                email: userProfile.user.email ?? null,
+                full_name: userProfile.user.full_name ?? null,
+                username: userProfile.user.username ?? null,
+              },
+              avatar,
```

Comment:
I think we want the `userProfile.data.avatar` here right to ensure that it's returned in the format we expect?